pyroSAR/_dev_config.py

```diff
@@ -154,7 +172,6 @@ def examine(name):
 
```

Comment:
wouldn't the folllowing work just as well:
```python
if isinstance(name, str):
    name = [name]
executable_list = list()
for item in name:
    executable_temp = find_executable(item)
    if executable_temp is not None:
        executable_list.append(os.path.abspath(executable_temp))
if len(executable_list) > 1:
    raise ValueError('Several SNAP instances were found: \n {}\n Please define your preference via pyroSAR.snap_config.set_path(...)'.format('\n'.join(executable_list))
elif len(executables_list) == 1:
    return True, executable_list[0]
else:
    warnings.warn("None of the following SNAP executables could be found by pyroSAR:\n {0}\n You can download it from http://step.esa.int/main/toolboxes/snap/ or you can specify a path with snap_config.set_path(path_to_snap)".format('\n'.format(name)), UserWarning)
```

Reply:
Good point. I cleaned the redundant parts. Thank you.